tests/components/zha/test_light.py

```diff
@@ -63,6 +67,39 @@
 }
 
 
+@asynctest.mock.patch(
+    "zigpy.zcl.clusters.general.OnOff.read_attributes", new=MagicMock()
+)
+async def test_light_refresh(hass, zigpy_device_mock, zha_device_joined_restored):
+    """Test zha light platform refresh."""
+
+    # create zigpy devices
+    zigpy_device = zigpy_device_mock(LIGHT_ON_OFF)
+    zha_device = await zha_device_joined_restored(zigpy_device)
+    on_off_cluster = zigpy_device.endpoints[1].on_off
+
+    # allow traffic to flow through the gateway and device
+    await async_enable_traffic(hass, [zha_device])
+    on_off_cluster.read_attributes.reset_mock()
+
+    # not enough time passed
+    async_fire_time_changed(hass, dt_util.utcnow() + timedelta(minutes=20))
+    await hass.async_block_till_done()
+    assert on_off_cluster.read_attributes.call_count == 0
```

Comment:
We should also assert entity state after each time interval.

Reply:
👍

Reply:
done